taskvine/src/bindings/python3/ndcctools/taskvine/dask_executor.py

```diff
@@ -127,6 +128,7 @@ def get(self, dsk, keys, *,
             progress_label="[green]tasks",
             wrapper=None,
             wrapper_proc=print,
+            prune_cluster=False,
```

Comment:
Let's call it `prune_files`.  I don't think that we explain the term `cluster` in the taskvine documentation, etc.